### DISCUSSION TITLE
How can I embed a PDF with MyST?

### DISCUSSION THREAD

**Discussion by emilopez** [executablebooks/meta#300] [Q&A]
Hi, I would like to embed a pdf file. 

I tried installing the [pdfembed sphinx extension]((https://github.com/SuperKogito/sphinxcontrib-pdfembed)).

I added in this in my `_config.yml `

```
sphinx:
  extra_extensions: [ 'sphinxcontrib.pdfembed' ]
```

and then in the md file

```
```{eval-rst}
:pdfembed:`src:_static/m.pdf, height:500, width:500, align:middle`
```
```
and copy m.pdf to `_static` directory,  finally I did  `jb build mini_book`. the build process completes without errors,  but then I cant see the pdf in the html.

I tried changing the path to the file, using `src:/_static/m.pdf`,  `src:./_static/m.pdf`,  but there was no case, it didn't work.
Any idea?
Thanks in advance.

**Comment by welcome** [comment]
Thanks for opening your first issue here! Engagement like this is essential for open source projects! :hugs:
<br>If you haven't done so already, check out [EBP's Code of Conduct](https://github.com/executablebooks/.github/blob/master/CODE_OF_CONDUCT.md). Also, please try to follow the issue template as it helps other community members to contribute more effectively.
<br>If your issue is a feature request, others may react to it, to raise its prominence (see [Feature Voting](https://executablebooks.org/en/latest/feature-vote.html)).
<br>Welcome to the EBP community! :tada:

**Comment by firasm** [comment]
Here's how I embed PDFs into my Jupyterbook:

```
<iframe src="../../2021-02-01 - Lecture 3.pdf" width="100%" height="800px" frameBorder="0"> </iframe>
```

[Here](https://firas.moosvi.com/courses/cosc111/class/week03/lecture.html) it is in action, the repository is open if you want to have a look at the full code and file path/structure.

Seems to work for me.

**Reply by emilopez** [reply]
Thanks! that worked for me, without using any extensions!

**Reply by fdion** [reply]
A neat advantage to using .* for extension is that you can then have, say, a `.png` and a `.pdf` and when using `html` builder it'll use the `png` and when using `pdflatex` as builder, it'll use the `pdf`

**Comment by choldgraf** [comment]
I'm not familiar with that Sphinx extension, but it does seem like it's doing something very similar to what @firasm describes, so I'd recommend just trying that.

For your specific question, I guess the extension is using page metadata to parameterize itself. You should try doing that with MyST Markdown rather than `eval-rst` like so:

````md
pdfembed: src:_static/m.pdf, height:500, width:500, align:middle
---
# My content

and so on...
````

**Comment by welcome** [comment]
Thanks for opening your first issue here! Engagement like this is essential for open source projects! :hugs:
<br>If you haven't done so already, check out [EBP's Code of Conduct](https://github.com/executablebooks/.github/blob/master/CODE_OF_CONDUCT.md). Also, please try to follow the issue template as it helps other community members to contribute more effectively.
<br>If your issue is a feature request, others may react to it, to raise its prominence (see [Feature Voting](https://executablebooks.org/en/latest/feature-vote.html)).
<br>Welcome to the EBP community! :tada:

**Reply by choldgraf** [reply]
Just a note that I converted this issue into a **discussion** here - we are trying to keep general questions / conversations / etc to the Discussions board, and to save the GitHub issues for items with actionable next steps related to a repository's codebase
